domains/raed.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "Raed",
```

provided username does not match github username
```suggestion
    "username": "raed667",
```